repository/p.json

```diff
@@ -217,6 +217,16 @@
 			]
 		},
 		{
+			"name": "Paste without blank lines",
```

Comment:
This line can be deleted, then the package will be listed as `PasteWithoutBlankLines`.